samples/dashboards/demo/cloud-monitoring/demo.html

```diff
@@ -0,0 +1,26 @@
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/modules/solid-gauge.js"></script>
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+
+<div class="cloud-monitoring-dashboard-container">
+  <div class="cloud-monitoring-data-controls">
+    <label for="enablePolling">
+      <input type="checkbox" id="enablePolling" name="enablePolling" />
+      <span>Enable data polling</span>
+    </label>
+  </div>
+  <div id="container"></div>
+</div>
+<p class="highcharts-description">
+  The demo shows how you can visualize AWS server data on a dashboard.<br/>
```

Comment:
As far as I remember we shouldn't use `<br/>` here

Reply:
In the minimal demo we use it as well.

Reply:
I agree with Karol. Use nested paragraphs instead. See https://github.com/highcharts/highcharts/blob/master/repo-guidelines.md#1-avoid-line-breaks-without-paragraphs .